dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/PersistenceExceptionMapper.java

```diff
@@ -0,0 +1,37 @@
+package io.dropwizard.hibernate;
+
+import org.hibernate.exception.DataException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.PersistenceException;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import javax.ws.rs.ext.Providers;
+
+@Provider
+public class PersistenceExceptionMapper implements ExceptionMapper<PersistenceException> {
```

Comment:
What do you mean by copy-paste? copy-paste the class name to register? Or copy-paste the contents?

If you're worried about informing users to register the exception mapper, maybe we can do something like we've done with [other exception mappers](https://github.com/dropwizard/dropwizard/blob/f7f1cc2966c59e0cf6e7b536f97737eab5481f40/dropwizard-core/src/main/java/io/dropwizard/setup/ExceptionMapperBinder.java) (we define default ones that can be overrode)